templates_generate.go

```diff
@@ -0,0 +1,53 @@
+// +build ignore
+
+package main
+
+import (
+	"log"
+	"net/http"
+	"os"
+	"time"
+
+	"github.com/shurcooL/vfsgen"
+)
+
+// zeroModTimeFileSystem is an http.FileSystem wrapper.
+// It exposes a filesystem exactly like Source, except
+// all file modification times are changed to zero.
+// See https://github.com/shurcooL/vfsgen/pull/40#issuecomment-355416103
```

Comment:
Why do we not want mtime?

Reply:
We do not need modified time because if the modified time changes but there is no content change, it would cause vfsgen to output a new file once more. This can lead to additional files in commits etc. Mod time isn't even relevant to our use case so I think this is the correct approach.

Reply:
But the new file would be of the same content hence where would not be a diff?

Reply:
this would probably save us from generating the same stuff again.

Reply:
If the mtime is the same why would it generate it again? If the mtime is different its likely different, if its not, there will be no diff hence nothing to commit.

Reply:
agreed. let's remove this